packages/vue/src/components/__tests__/confirm-reset-password.spec.ts

```diff
@@ -135,14 +135,24 @@ describe('ConfirmResetPassword', () => {
     expect(updateBlurSpy).toHaveBeenCalledWith({ name: 'confirm_password' });
   });
 
-  // @todo-migration fix?
-  it.skip('sends submit event on form submit', async () => {
+  it('sends submit event on form submit', async () => {
     render(ConfirmResetPassword, { global: { components } });
-
+    // we need to first fill out the form
+    const codeField = await screen.findByLabelText('Code *');
+    const newPasswordField = await screen.findByLabelText('New Password');
+    const confirmPasswordField =
+      await screen.findByLabelText('Confirm Password');
+    await fireEvent.input(codeField, { target: codeInputPrams });
+    await fireEvent.input(newPasswordField, {
+      target: newPasswordInputParams,
+    });
+    await fireEvent.input(confirmPasswordField, {
+      target: confirmPasswordInputParams,
+    });
+    // then we can submit the form
```

Comment:
Curious why we need so much additional code in this test? Same question applies to other unit test in this PR

Reply:
My thinking was since basically forms can only be submitted if they are filled out properly, so first we need fill the form. I can have a second look, just to double check if you think that it's a blocker.

Reply:
What changed that caused the need for the additional code?

Reply:
That test was failing because it was testing the action of submitting the form, that's why I came up with this approach, so the form can only be submitted after all the fields have been filled out first.

Reply:
Previous to the MV, this test passed without the additional actions in the unit test. I am curious as to what caused the unit test to no longer pass, which will give the required context to determine if the changes made here are appropriate

Reply:
I looked at the migration commit and the Vue package doesn't seem to be affected by much, only a few dependency updates, so the current solution that passes all the tests is the only one I could come up with.

Reply:
The open question is still why this test requires so many changes, where as other Vue Authenticator test files with the same test did not. Not looking for an "optimized" solution here, looking to understand the need for this solution since it differs from the aforementioned test files